src/client/WindshaftMetadata.js

```diff
@@ -99,7 +99,9 @@ export default class WindshaftMetadata extends MVTMetadata {
         const column = this.properties[baseName];
         let dimension = null;
         if (baseName !== propertyName) {
-            if (column.dimension) {
+            if (baseName !== propertyName && column.dimension) {
+                // whatever
```

Comment:
👀 I'm afraid I didn't check my suggestion here <<

Reply:
:joy: I noticed after accepting the suggestion